boards/arm/faze/faze-pinctrl.dtsi

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022 NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/lpc11u6x-pinctrl.h>
+
+&pinctrl {
+	pinmux_uart0: pinmux_uart0 {
```

Comment:
suggestion: name using target state, e.g. `uart0_default`, `i2c0_fast`, etc.

Reply:
Fixed to use target state naming